lib/ansible/utils/display.py

```diff
@@ -412,7 +417,12 @@ def display(
             #         raise
 
         if logger and not screen_only:
-            msg2 = nocolor.lstrip('\n')
+            self.log(nocolor, color)
+
+    def log(self, msg: str, color: str | None = None, caplevel: int = 0):
```

Comment:
We're technically making new public API here, so we should probably either give the args some more careful thought, or `_` prefix the `log` method. Also, direct post-fork callers to `log` would bypass the display proxy, requiring shared access to the logger and its associated file handles, which works against recent efforts to minimize post-fork shared file handles. I thought we'd already proxied `verbose`, but maybe not- we should probably just `_` prefix the new method and make sure it can only be called pre-fork.